community.rst

```diff
@@ -1,3 +1,5 @@
+.. title:: Community
```

Comment:
What is this ".. title::" part? I've never seen this before. Do you have a link to a reference page about it?

Reply:
I put it in there, to be able to link there from the new page.

Reply:
I copied it from `gallery.rst` where it is also done.

Reply:
I believe the usual way of doing this in `rst` is ```:doc:`community` ```. I'll look at the gallery page and see if I can figure out why it is done. I can't find documentation on this title identifier.

Reply:
Thanks, let me know, what I should change.

Reply:
It looks like you are already doing the `doc` linking properly, but this title is optional. As mentioned [here](http://docutils.sourceforge.net/docs/ref/rst/directives.html#metadata-document-title) it forces the "meta" title for things like HTML building (other writers may use it differently). So in the case of HTML it would for the title of the web page in the title/tab bar. I'd say remove it reduce confusion. Similarly remove it from the raspberry page too unless you would really like the HTML title to be different than `rst` title (unnecessary IMO).